### PR TITLE
make the callback plugin accept other URL env vars

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: plugins/callback/foreman.py [hunk 1]
```diff
@@ -20,16 +20,20 @@
       - requests (python library)
     options:
       url:
-        description: URL to the Foreman server
+        description:
+          - URL of the Foreman server.
         env:
           - name: FOREMAN_URL
+          - name: FOREMAN_SERVER_URL
+          - name: FOREMAN_SERVER
```
Comment (mdellweg) [inline plugins/callback/foreman.py:28]: This looks like we'd want all three combinations in the other places too, right?

Comment (evgeni) [reply plugins/callback/foreman.py:28]: preferably not? but I can be convinced ;)

Comment (mdellweg) [reply plugins/callback/foreman.py:28]: :cyclone: _convince_ :cyclone:

Comment (evgeni) [reply plugins/callback/foreman.py:28]: very convincing tornado

Comment (mdellweg) [reply plugins/callback/foreman.py:28]: for consistency!

**Review by mdellweg — APPROVED** [review]

**Pull request merged**